apps/partners-reference/cypress/integration/navigation.ts

```diff
@@ -0,0 +1,23 @@
+describe("Navigating around the site", () => {
+  it("Loads the homepage directly", () => {
+    cy.visit("/")
+    // Check if sign in prompt is displayed
+    cy.contains("Sign In")
+  })
+
+  it("Signs in", () => {
+    cy.visit("/")
+    cy.fixture("user").then((user) => {
+      cy.get("input#email").type(user.email)
+      cy.get("input#password").type(user.password)
+      cy.get(".button.is-filled").contains("Sign In").click()
+      cy.contains("This will be the home page")
+    })
+  })
+
+  it("Applications page", () => {
+    cy.visit("/")
```

Comment:
This is going to fail if the user hasn't already logged in - it's dependent on state from the previous test (logging in) to succeed. More broadly speaking, since we require login for basically all pages on the partners app, a lot of the tests are going to be dependent on login. I think we should extract that into a helper function that can be run `beforeEach` in order to allow us to decouple tests/run them in isolation.

We can probably do this in 2 ways:

1) Extract the login logic in the previous test into a helper function.
2) Don't rely on the UI - make a request to the API programmatically with Cypress to get an access token, then write it to localStorage before the rest of the test runs. This is probably better fitting with Cypress best practices. I can post a commit of what I'm thinking...